README.md

```diff
@@ -87,21 +87,21 @@ A minimal example of using an existing configuration.nix:
 
   outputs = { self, darwin, nixpkgs }: {
     darwinConfigurations."Johns-MacBook" = darwin.lib.darwinSystem {
-      system = "x86_64-darwin";
       modules = [ ./configuration.nix ];
     };
   };
 }
 ```
 
-Inputs from the flake can also be passed to `darwinSystem`. These inputs are then
+Inputs from the flake can also be passed into `darwinSystem`. These inputs are then
 accessible as an argument `inputs`, similar to `pkgs` and `lib`, inside the configuration.
 
 ```nix
 darwin.lib.darwinSystem {
-  system = "x86_64-darwin";
-  modules = [ ./configuration.nix ];
-  inputs = { inherit darwin dotfiles nixpkgs; };
+  modules = [
+    ./configuration.nix
+    { _module.args.inputs = { inherit darwin dotfiles nixpkgs; }; }
```

Comment:
```suggestion
  modules = [ ./configuration.nix ];
  specialArgs = { inputs = { inherit darwin dotfiles nixpkgs; }; };
```

I think we should direct them to use `specialArgs`. It could be worthwhile to rewrite this section as `inputs` is no longer an input to `darwinSystem`, possibly showing the entire flake and showing how to get all the flake inputs as `inputs`.

Reply:
Agreed. I will change accordingly.